spreadsheet/workbook.go

```diff
@@ -263,7 +263,9 @@ func (wb *Workbook) Save(w io.Writer) error {
 		fmt.Println("Unlicensed version of UniOffice")
 		fmt.Println("- Get a license on https://unidoc.io")
 		for _, sheet := range wb.Sheets() {
-			a1 := sheet.Cell("A1")
+			row1 := sheet.Row(1)
+			row1.SetHeight(50)
+			a1 := row1.Cell("A")
```

Comment:
what is the point of this?  Change lineheight?

Reply:
Yes, sometimes the message looks obscured, I did it to make it more visible